src/index.js

```diff
@@ -0,0 +1,89 @@
+import { editTodo, removeTodo } from './edit-remove.js';
+import { getFromLocal, updateLocalStorage } from './local-storage';// eslint-disable-line
+import './style.css';
+
+// Queries to HTML
+const inputText = document.querySelector('input');
+const todoMainContainer = document.querySelector('.todos-container');
+const ClearBtn = document.querySelector('button');
+
+// class object
+class MyObjects {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] Please kindly do not disable linters, fix linters errors helps programmers to write clean and clear codebase 😁 😉